components/usage/pkg/db/workspace_instance.go

```diff
@@ -80,6 +80,8 @@ func FindRunningWorkspaceInstances(ctx context.Context, conn *gorm.DB) ([]Worksp
 	tx := queryWorkspaceInstanceForUsage(ctx, conn).
 		Where("wsi.stoppingTime = ?", "").
 		Where("wsi.usageAttributionId != ?", "").
+		// Exclude anything before August 2022, we cannot guarantee data quality before then
+		Where("wsi.startedTime > ?", TimeToISO8601(time.Date(2022, 8, 1, 0, 0, 0, 0, time.UTC))).
```

Comment:
Question: Wasn't this part of why we have a `billInstancesAfter`? 💭

Reply:
`billInstancesAfter` would be a check against the `effectiveTime`. Running Instances have their `effectiveTime` now or in the future.

Reply:
The `billInstancesAfter` is/was used in the BillingService. We could re-use it here as well

Reply:
I don't think so. It should actually be named `billInstancesStoppedAfter` to make clear it has nothing to do with when a workspace was started.

Reply:
I think we should remove `billInstancesStoppedAfter` entirely. Doesn't make sense to have a global starting point. Instead we should have the information how usage gets billed associated with each cost center. I'm going to work on this in the context of https://github.com/gitpod-io/gitpod/issues/12741